src/pages/about/aboutView.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
-import { AboutHeader, AboutBody } from '@bento-core/about';
+import AboutHeader from '../../components/about/aboutHeaderView';
```

Comment:
@AjayDoddapaneni Can we use AboutHeader & AboutBody from @bento-core/about instead of local code?